sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/OffHeapColumnVector.java

```diff
@@ -42,6 +43,27 @@ public static OffHeapColumnVector[] allocateColumns(int capacity, StructType sch
     return allocateColumns(capacity, schema.fields());
   }
 
+  /**
+   * Allocates columns to store elements of each field of the schema,
+   * the constant column use `ConstantColumnVector` and others use `OffHeapColumnVector`,
+   * this method assumes that all constant column are at the end of schema.
```

Comment:
Can we make this clearer? I think it is a very important assumption. For example, you can put this on a new line to emphasise significance.

Reply:
ok